### PR TITLE
One life event version for death and burial

### PR DESCRIPTION
**Pull request by lgarceau768**

### CODE REVIEW

**Comment by lgarceau768** [comment]
@flexyjesse @fpigeonjr Ready for review!

**Review by fpigeonjr — APPROVED** [review]
Today's demo was from this branch and it worked well!

**Comment by fpigeonjr** [comment]
this includes #389 so it can be closed once this is merged.

**Review by r-bartlett-gsa — APPROVED** [review]

File: jest.setup.js [hunk 1]
```diff
@@ -6,3 +6,6 @@ const translations = nuxtConfig.i18n.vueI18n.messages
 
 config.mocks.$t = (msg) => translations[locale][msg]
 config.mocks.switchLocalePath = (msg) => msg
```
Comment (jschoenemann-cms) [inline jest.setup.js:8]: Should move this into the specific test.

**Review by jschoenemann-cms — APPROVED** [review]
Prob could have broken up this into a couple PRs.  Date input stuff could have been removed from one life event stuff.  Not the end of the world though.  Good job. 

Approved, please make that one small change though.

**Pull request merged**